.github/workflows/test.yml

```diff
@@ -49,3 +49,20 @@ jobs:
       run: |
         cd docs
         make doctest
+
+  test-paper-code:
+  runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[full]
+    - name: Test with pytest
+      run: |
+        cd docs
+        make doctest
```

Comment:
Why don't we just check that the paper code snippets run?

Reply:
That is not the up-to-date version. The following is the new version
```
  test-paper-code:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Set up Python
      uses: actions/setup-python@v2
      with:
        python-version: 3.9
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
        pip install .[full]
    - name: Test with pytest
      run: |
        cd docs/paper
        mkdir figures
        pytest *.py
```
I need to make a figures directory so that `figsave("figures/...")` works. But maybe `mkdir` is not allowed in GitHub action. Maybe I'll just remove the subfolder `figures` and just use the current folder.